kubetools/deploy/image.py

```diff
@@ -84,7 +88,10 @@ def _ensure_docker_images(
     kubetools_config, build, app_dir, commit_hash,
     default_registry=None,
     check_build_control=lambda build: None,
+    additional_tags=None,
 ):
+    if additional_tags is None:
+        additional_tags = []
```

Comment:
Could we just take in `additional_tags` as an empty list as kwarg default? Or is that unpythonic of me

Reply:
In this case that would be fine as it is read only. I went this way out of habit, as mutable defaults are a pain when they are mutated in the function.